acl2020_tools/awscognito/create_user_lambda.py

```diff
@@ -0,0 +1,45 @@
+# 
+# This lambda function uses `email` and `name` from the event
+# to create a user in the 'attendees' user group in Cognito
+# pool in the same region
+#
+# Configuration:
+#  ACL2020_USER_POOL_ID - environment variable specifying 
+#
+# IAM policies for the lambda's role:
+#  AWSLambdaBasicExecutionRole - required by default
+#  AmazonCognitoPowerUser - yes, not that secure, but works for test  
+# 
+# Runtime:
+#   Python 3.7                                  
+
+import os
+import json
+import boto3
+from dataclasses import dataclass
+import cognito  # type: ignore
```

Comment:
this means you need to upload this `cognito.py` file to the Lambda function, right?
Did you check if it works properly? (I guess so, just want to make sure there is no import issue..)

Reply:
Yes for the both questions :)